### DISCUSSION TITLE
Internet Explorer detected

### DISCUSSION THREAD

**Discussion by KozRipefruit** [bootscore/bootscore#345] [Q&A]
I have no issue with checking for Internet Explorer, except the message has appeared in my CHROME browser.

How can I tweak / disable it?

**Comment by crftwrk** [comment]
The alert message appears in Chrome? Do you have a link to check?

For now, you can just delete IE-alert in main-theme `js/theme.js` in line `43`. There you will find this:

```js
// IE Warning
if (window.document.documentMode) {
  let IEWarningDiv = document.createElement('div');
  IEWarningDiv.setAttribute('class', 'position-fixed top-0 end-0 bottom-0 start-0 d-flex justify-content-center align-items-center');
  IEWarningDiv.setAttribute('style', 'background:white;z-index:1999');
  IEWarningDiv.innerHTML = '<div style="max-width: 90vw;">' + '<h1>' + bootscore.ie_title + '</h1>' + '<p className="lead">' + bootscore.ie_limited_functionality + '</p>' + '<p className="lead">' + bootscore.ie_modern_browsers_1 + bootscore.ie_modern_browsers_2 + bootscore.ie_modern_browsers_3 + bootscore.ie_modern_browsers_4 + bootscore.ie_modern_browsers_5 + '</p>' + '</div>';
  document.body.appendChild(IEWarningDiv);
}

```

Comment or delete it. 

You can also add a unique class like `ie-warning-alert` to

```js
IEWarningDiv.setAttribute('class', 'ie-warning-alert position-fixed top-0...'
```

and hide message by css:

```css
.ie-warning-alert {
  display: none !important;
}
```

Want to create a PR?

**Reply by KozRipefruit** [reply]
sure... message still there today 
https://www.fmhc.com.au/

**Reply by KozRipefruit** [reply]
I'll leave it there until you have checked it

**Reply by crftwrk** [reply]
I see no alert in Chrome? You are using an earlier version, there it is different as described above. Open `header.php` in your child and scroll down to the bottom. There you will find this:

`<?php bootscore_ie_alert(); ?>`

Delete it.

Solved?

**Reply by KozRipefruit** [reply]
No-one was more surprised than me
Latest Version 108.0.5359.125 (Official Build) (64-bit)
I commented the code in the js file
Thanks for your assistance
